pkg/executor/windows/builder.go

```
@@ -63,10 +64,12 @@ func Build(sctx sessionctx.Context, v *physicalop.PhysicalWindow, childExec exec
 		}
 		agg := aggfuncs.BuildWindowFunctions(exprCtx, aggDesc, resultColIdx, orderByCols)
 		windowFuncs = append(windowFuncs, agg)
-		partialResult, _ := agg.AllocPartialResult()
+		partialResult, memDelta := agg.AllocPartialResult()
 		partialResults = append(partialResults, partialResult)
+		initialPartialResultMemUsage += memDelta
 		resultColIdx++
 	}
+	memTracker := newWindowMemoryTracker(initialPartialResultMemUsage, len(windowFuncs))
```

golangci-lint (2.12.2)

[warning] line-length-limit: line is 88 characters, out of limit 80

(revive)